### ISSUE TITLE
BROWSER vs NODE differences

### ISSUE THREAD

**Issue by oskrs111** [jDataView/jBinary#63]
Hi,
I'm trying to port a working app from electron (node based) to cordova. This app makes intesive usage of jBinary and works really fine on electron.

What i found is that BROWSER version does not generate the same jBinary::view type as when using NODE version. For the the same code;

    var writeBuffer = new Buffer(2048);
    var writeFrame = new jBinary(writeBuffer, bdFrameStruct);
    
NODE version -> 
*var jBinary = require('jbinary'); * is generating view as 'jDataView'

> writeFrame
>jBinary {**view: jDataView**, contexts: Array[0], typeSet: Object, cacheKey: "jBinary.Cache.1"}
 
BROWSER version -> 
*<script type="text/javascript" src="js/jdataview.js"></script>*
*<script type="text/javascript" src="js/jbinary.js"></script>*  is generating view as 'e'

> writeFrame
> h {**view: e**, contexts: Array[0], typeSet: Object, cacheKey: "jBinary.Cache.1"}

How do can I get the same result as in NODE using BROWSER?
Or maybe there is a better way to use it on cordova?

Thanks.

**Comment by RReverser** [comment]
It should be still jDataView, just the name is minified. Do you have any difference in actual behaviour?

**Comment by oskrs111** [comment]
Yes, that's the problem. NODE version works perfectly but when switching to BROWSER does not. Note that when I say BROWSER I am still using electron only switching to js file version instead node module, It is supposed to work the same  Insn't it?

Here is what i have found going deeper into JBinary Object. Same piece of code, expanding 'view'

    var writeBuffer = new Buffer(2048);
    var writeFrame = new jBinary(writeBuffer, bdFrameStruct);

NODE VERSION -->
>writeFrame.view
jDataView {**buffer: Buffer[2048]**, _isArrayBuffer: false, _isPixelData: false, _isDataView: false, **_isNodeBuffer: true**…}

Note-> **buffer: Buffer[2048]**
Note-> **_isNodeBuffer:true**

BROWSER VERSION ->
>writeFrame.view
>e {**buffer: ArrayBuffer**,** _isArrayBuffer: true**, _isPixelData: false, **_isDataView: true**, _isNodeBuffer: false…}

Note-> **buffer: ArrayBuffer**
Note-> **_isArrayBuffer: true**
Note-> **_isDataView: true**

Any idea? Why does 'buffer' type is also different?

**Comment by RReverser** [comment]
Well `buffer` being different is expected, it's an internal platform-specific representation which you shouldn't use anyway in most cases. That's because jDataView chooses the best buffer type depending on what's supported in the current engine.

But, as you can see, it's still jDataView in both cases, so all the public APIs should still work.
